src/main/java/org/sopt/confeti/api/user/facade/UserTimetableFacade.java

```diff
@@ -168,8 +168,11 @@ public UserTimetableFestivalBasicDTO getTimetableInfo(final long userId, final l
                 .collect(Collectors.toMap(userTimetable ->
                         userTimetable.getFestivalTime().getId(), Function.identity())
                 );
-
-        return UserTimetableFestivalBasicDTO.of(festivalDate, userTimetableMapper);
+        try {
+            return UserTimetableFestivalBasicDTO.of(festivalDate, userTimetableMapper);
+        } catch (NullPointerException e) {
+            throw new NotFoundException(ErrorMessage.NOT_FOUND);
+        }
```

Comment:
P2. `NPE` 발생 시 외부에서 404 응답 에러 처리를 해주셨군요. 잘 작성하신 것 같아요.
다만, 현재 DTO에서 값의 존재 유무를 에러로 확인하는 책임이 있어서 이걸 Mapper 변수를 생성하는 파사드 계층으로 가져오면 좋을 것 같아요.

```suggestion 
        validateExistFestivalTimeIds(userTimetableMapper, festivalTimeIds);
        return UserTimetableFestivalBasicDTO.of(festivalDate, userTimetableMapper);
```

Reply:
P3. 추가로 현재 파사드 계층에서 `ArtistResolver`의 `load` 함수를 호출하는 것으로 보여져요.
아티스트 정보 주입 계층은 서비스 계층으로 통일되어있어 수정해주시면 좋을 것 같아요.

Reply:
P3. 추추가로 저희 코드 컨벤션 내용 중 `메서드 내부 최대 길이 중괄호 포함 하지 않고 15줄까지 허용`이 있어요. 최대한 바보 같은 코드를 만들기 위해 어떻게 분리할지 고민해보시면 좋을 것 같습니다!

Reply:
P4. 현재 `@Min` 어노테이션이 동작하지 않는 것 같아요. 저도 해결방법을 찾는 중이에요. 같이 알아보고 혹시라도 먼저 찾으시면 연락 부탁드릴게요!

Reply:
리뷰 반영해 아래와 같이 수정해 두었습니다! 
DTO까지 도달하기 전에 값을 검증할 수 있도록 validateExistFestivalTimeIds와 validateExistUserTimetableMapper를 추가했습니다. 또, userTimetableMapper 생성 로직을 createUserTimetableMapper 메서드로 분리하여 코드 가독성을 높여 보았습니다!

```java
 @Transactional(readOnly = true)
    public UserTimetableFestivalBasicDTO getTimetableInfo(final long userId, final long festivalDateId) {
        validateUserExists(userId);

        FestivalDate festivalDate = festivalDateService.findFestivalDateId(festivalDateId);

        List<Long> festivalTimeIds = festivalDate.getStages().stream()
                .flatMap(festivalStage -> festivalStage.getTimes().stream())
                .map(FestivalTime::getId)
                .toList();
        validateExistFestivalTimeIds(festivalTimeIds);

        List<UserTimetable> userTimetables = userTimetableService.getUserTimetables(userId, festivalTimeIds);

        Map<Long, UserTimetable> userTimetableMapper = createUserTimetableMapper(userTimetables);
        validateExistUserTimetableMapper(userTimetableMapper);

        return UserTimetableFestivalBasicDTO.of(festivalDate, userTimetableMapper);
    }
```

말씀해 주신 대로 바보 같은 코드를 작성하기 위해 앞으로 더 고민이 필요할 것 같네요...  꼼꼼한 리뷰 감사드려요! 👍🏻